### ISSUE TITLE
Kodi nexus 

### ISSUE THREAD

**Issue by RosensRauk** [basilgello/Transmission-Control-Panel#3]
Is this addon supposed to work on nexus?
I'm having troubles getting the add-on to work on Kodi nexus installed on a fire cube 2nd gen.
Transmission is on a local server and I can connect to it from within the network.

**Comment by basilgello** [comment]
Yes, just checked on Nexus 2:20.0+dfsg-1 from Debian bookworm. Can you please describe your set up and provide Kodi debug log?
